x-pack/platform/plugins/shared/alerting/server/usage/lib/get_telemetry_from_alerts.ts

```diff
@@ -48,6 +49,13 @@ export async function getTotalAlertsCountAggregations({
             field: 'kibana.alert.rule.rule_type_id',
             size: NUM_ALERTING_RULE_TYPES,
           },
+          aggs: {
+            ignored_field: {
+              terms: {
+                field: '_ignored',
```

Comment:
you may need to set a size field for the term aggregation. the default is 10 so if there ends up being more than 10 rule types with ignored fields, this won't return all of them.